src/main/kotlin/com/hibob/project/resource/feedbackResource.kt

```diff
@@ -0,0 +1,36 @@
+package com.hibob.project.resource
+
+import com.hibob.project.dao.*
+import com.hibob.project.service.FeedbackService
+import com.hibob.project.utils.AuthenticationUtil
+import jakarta.ws.rs.*
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.core.Context
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+
+@Controller
+@Path("/api/system/feedback")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class FeedbackResource(private val feedbackService: FeedbackService) {
+    @GET
+    fun getAllFeedbacks(@Context requestContext: ContainerRequestContext): Response {
+        val authenticatedEmployee = AuthenticationUtil.extractAuthenticatedEmployee(requestContext)
+        if (authenticatedEmployee.role == Role.ADMIN || authenticatedEmployee.role == Role.HR) {
+            val feedbacks = feedbackService.getAllFeedbacks(authenticatedEmployee.companyId)
+            return Response.ok(feedbacks).build()
+        }
+
+        return Response.ok().build()
```

Comment:
why is the response "ok"? if we reached this code part it means we didn't have permission